src/buttons/RolesEditionButton/components/RolesEditionDialog/RolesEditionDialog.js

```diff
@@ -0,0 +1,432 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  Grid,
+  TextField,
+  DialogActions,
+  Button,
+  IconButton,
+  Typography,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+  Divider,
+  Chip,
+  makeStyles,
+} from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { RoleEditor } from '../../../../inputs';
+import PersonIcon from '@material-ui/icons/Person';
+import DesktopMacIcon from '@material-ui/icons/DesktopMac';
+
+const useStyles = makeStyles((theme) => ({
+  dialogActions: {
+    marginRight: '4px',
+    marginBottom: '4px',
+  },
```

Comment:
Why do you define margins here? is it because they are different on the mockups? I'd rather let MUI define margins according to Material Design specifications :)